src/Microsoft.ML/MemoryCollection.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+
+namespace Microsoft.ML
```

Comment:
>.ML [](start = 19, length = 3)

ML.Data

Reply:
TextLoader is part of just ML, should I change it as well?

---
In reply to: [187221578](https://github.com/dotnet/machinelearning/pull/106#discussion_r187221578) [](ancestors = 187221578)

Reply:
good question... seems they both should be in data. the argument went about like this - can a user has out of the box experience with just ML namespace.  
I guess we can keep it in ML for now.

---
In reply to: [187221799](https://github.com/dotnet/machinelearning/pull/106#discussion_r187221799) [](ancestors = 187221799,187221578)

Reply:
Please move to Data. My PR will move TextLoader to ML.Data.